input/fsh/Clinical datapoints profiles/ClinicalFactors_BodyMeasures.fsh

```diff
@@ -0,0 +1,27 @@
+ValueSet: UnitsBodyHeightVS
+Id: UnitsBodyHeight
+Title: "Units of patient's body height"
+Description: "Valueset of the unit  of the patient's body height"
+* SCT#258672001 "Centimeter" 
+* SCT#258677007 "Inch"
+
+ValueSet: UnitsBodyWeightVS
+Id: UnitsBodyWeight
+Title: "Units of patient's body weight"
+Description: "Valueset of the unit  of the patient's body weight"
+* SCT#258683005 "kilogram"
+* SCT#258693003 "pounds"
```

Comment:
These should actually be [UCUM units](https://ucum.org/ucum.html), not snomed ones - per "[If the unit can be coded in UCUM and a code is provided, it SHOULD be a UCUM code](https://www.hl7.org/fhir/datatypes.html#Quantity)".

Reply:
I wasn't aware of this, I just used the snomed codes provided by Linda. I'll just completely reuse the vital signs standard profile for body weight and body height